aten/src/ATen/native/cuda/CuFFTPlanCache.h

```diff
@@ -346,6 +346,7 @@ class CuFFTConfig {
 //     be fine for now.
 // TODO: When CUDA 10 comes out, check if the bug is fixed or if we need another
```

Comment:
The obsolete comment should be removed.

Reply:
That sounds good! I'll remove the comment related to the bug once I have a number for CUDA 10 and will add a line why 1023 has to be the max number when building without CUDA 10, if that's reasonable.

Reply:
Sounds great! Thanks!